app/src/main/java/com/divyanshu/splitter/MainActivity.kt

```diff
@@ -20,28 +21,17 @@ class MainActivity : ComponentActivity() {
         setContent {
             SplitterTheme {
                 Scaffold(modifier = Modifier.fillMaxSize()) { innerPadding ->
-                    Greeting(
-                        name = "Android",
-                        modifier = Modifier.padding(innerPadding)
-                    )
+                    Column(
+                        horizontalAlignment = Alignment.CenterHorizontally,
+                        verticalArrangement = Arrangement.Center,
+                        modifier = Modifier.fillMaxSize().padding(innerPadding)
+                    ) {
+                        Text(
+                            text = "Welcome to Splitter!",
+                        )
+                    }
                 }
             }
         }
     }
 }
-
-@Composable
-fun Greeting(name: String, modifier: Modifier = Modifier) {
-    Text(
-        text = "Hello $name!",
-        modifier = modifier
-    )
-}
-
-@Preview(showBackground = true)
```

Comment:
Why are you removing preview. I dont understand ;)

Reply:
Added